scenarios/aws/microVMs/microvms/libvirt_fs.go

```diff
@@ -130,7 +130,7 @@ func NewLibvirtFSDistroRecipe(ctx *pulumi.Context, vmset *vmconfig.VMSet, pools
 		pools:         pools,
 		volumes:       volumes,
 		baseVolumeMap: baseVolumeMap,
-		fsNamer:       libvirtResourceNamer(ctx, vmset.Name),
+		fsNamer:       libvirtResourceNamer(ctx, vmset.Tags...),
```

Comment:
should this be `vmset.ID`?

Reply:
It can be used here. But since, there is no need to use the arch in the namer, I would rather avoid it.

Reply:
Why is `arch` not necessary here?

Reply:
Kind of incidental, but the reason is that the resources this namer is used to generate all use the `command.Runner` provided by the underlying test-infra framework. This appends to the URN some string generated from the name of the ec2 VM, which contains the `arch`